src/main/java/com/taboola/backstage/model/dynamic/DynamicFieldDataType.java

```diff
@@ -13,6 +13,7 @@
     PERCENT,
     NUMERIC,
     DATA,
+    DATE,
```

Comment:
@vladimanaev while we you are here ? did you check nothing else it missing ?

Reply:
I went over the list, those are the main ones rest are not in use.